apps/modernization-ui/src/apps/search/patient/add/layout/AddPatientLayout.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactNode } from 'react';
+import styles from './add-layout.module.scss';
+import { AddPatientSideNavigation } from './sideNavigation/AddPatientSideNavigation';
+import { AddPatientHeaderContent } from './headerContent/AddPatientHeaderContent';
+import { AddPatientInPageNavigation } from './inPageNavigation/AddPatientInPageNavigation';
+import { NavSection } from 'design-system/inPageNavigation/InPageNavigation';
+
+interface AddPatientLayoutProps {
+    headerActions: ReactNode;
+    headerTitle: string;
+    patientForm: ReactNode;
```

Comment:
Question: Why is the `patientForm` being passed to this component? Is the parent to this going to handle passing down either the short or extended form?

Reply:
yes the parent is going to handle passing down either the short or extended form to the layout same as search layout currently passes down the search criteria. That way AddPatientLayout is focused on layout and structure, not on specifics of the form content and makes it a bit more generic

Reply:
after talking a bit with MP, Im going to change the current flow of data with the component accepting three props like so and setting AddPatientLayout to take just a node to be the form and inPageNav. 
This will allow the parent to pass either basic or extended forms based on the state.